src/main/java/com/github/markusbernhardt/seleniumlibrary/locators/WindowManager.java

```diff
@@ -45,45 +45,24 @@ public void select(WebDriver webDriver, SelectCoordinates selectCoordinates) {
 
 			@Override
 			public void select(WebDriver webDriver, final SelectCoordinates selectCoordinates) {
-				selectMatching(webDriver, new Matcher() {
-
-					@Override
-					public boolean match(List<String> currentWindowInfo) {
-						return currentWindowInfo.get(WINDOW_INFO_INDEX_DOCUMENT_TITLE).trim().toLowerCase()
-								.equals(selectCoordinates.criteria.toLowerCase());
-					}
-
-				}, "Unable to locate window with title '" + selectCoordinates.criteria + "'");
+				selectMatching(webDriver, currentWindowInfo -> currentWindowInfo.get(WINDOW_INFO_INDEX_DOCUMENT_TITLE).trim().toLowerCase()
+						.equals(selectCoordinates.criteria.toLowerCase()), "Unable to locate window with title '" + selectCoordinates.criteria + "'");
```

Comment:
Could you split the arguments and make it more readable, please?

Reply:
And for the text 2 methods also...

Reply:
Done